src/Ulid.php

```diff
@@ -87,4 +92,58 @@ public function __toString()
     {
         return $this->string();
     }
+
+    //"0123456789ABCDEFGHJKMNPQRSTVWXYZ"
+    protected static function crockford()
+    {
+        static $base32;
+
+        if (!isset($base32)) {
+            $base32 = new \Tuupola\Base32([
+                "characters" => \Tuupola\Base32::CROCKFORD,
+                "padding" => false
+            ]);
+        }
+        return $base32;
+    }
+
+    public static function sanitizeString($value)
```

Comment:
This should be a separate PR. As a sidenote it is not 100% clear if ULID spec requires this even though is is part of Crockford standard.